### PR TITLE
Add jQuery, Next.js & Bootstrap resources for courses in couses-hi.md

### PR DESCRIPTION
**Pull request by kanikamalhans**
## What does this PR do?
Add resource(s)

### Description
Added free courses in Hindi for Jquery, Next js & Bootstrap

### Why is this valuable (or not)?
These are great resources in local language for a wider audience

### How do we know it's really free?
ALl of them are on youtube & complete

### For book lists, is it a book? For course lists, is it a course? etc.
These are courses
## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by eshellman** [comment]
Don't invent titles! Use the title given by the resource.

File: courses/free-courses-hi.md [hunk 1]
```diff
@@ -213,6 +213,7 @@
 #### Bootstrap
 
 * [Bootstrap 3 Tutorial](https://youtube.com/playlist?list=PLjVLYmrlmjGciJ5_Ze6jDIfJpEeMYtsSe) - WsCube Tech
+* [Bootstrap Tutorial](https://www.youtube.com/playlist?list=PL0b6OzIxLPbz1cgxiH5KCBsyQij1HsPtG) - Yahoo Baba
```
Comment (shrn01) [inline courses/free-courses-hi.md:216]:
```suggestion
* [Bootstrap Tutorial for beginners in Hindi / Urdu](https://www.youtube.com/playlist?list=PL0b6OzIxLPbz1cgxiH5KCBsyQij1HsPtG) - Yahoo Baba
```

File: courses/free-courses-hi.md [hunk 2]
```diff
@@ -256,10 +257,12 @@
 * [jQuery Tutorial for beginners in Hindi 2020](https://youtube.com/playlist?list=PL-6FWL4WVVWXmWe_HnPG0rBQmmJfGsTKS) - CSEtutorials
 * [jQuery Tutorial in Hindi](https://youtube.com/playlist?list=PLvQjNLQMdagPRDnMQPMs5-vQL_IyAB0St) - Teaching Web
 * [jQuery Tutorials in Hindi 2018](https://youtube.com/playlist?list=PLwGdqUZWnOp0X4dVwSsEd6dV49TLLCooI) - Vinod Bahadur Thapa (Thapa Technical)
+* [jQuery Video Course](https://www.youtube.com/playlist?list=PLPAcs2twrK5_FzWMttkbTuM0dB5A-ni-Q) - w3webschool.net
 
 
 #### Next.js
 
+* [Next.js Tutorial in Hindi](https://www.youtube.com/playlist?list=PLolI8AY2AS9buXWmF0kLeBg6ETST4it1) - Code Improve
```
Comment (shrn01) [inline courses/free-courses-hi.md:265]: This link is broken for me

Comment (shrn01) [inline courses/free-courses-hi.md:265]:
```suggestion
```

**Pull request merged**